inference-engine/src/gna_plugin/frontend/scale_factor_calc.hpp

```diff
@@ -527,7 +529,7 @@ class ScaleFactorPerLayer<InferenceEngine::ConcatLayer*> {
                     gnalog() << "[UFS] from : " << concatLayer->name << " reached: " << layer->name;
                     // found that direct input to concat is a indirect parent of align filter - so no link required
                     auto info = LayerInfo(layer);
-                    if (!info.isWeightable() && !info.isActivation()) {
+                    if (!info.isWeightable() && !info.isActivation() && !info.isConst()) {
```

Comment:
requantizing const might be desirable, especially in cases that there are only input and concat layers.